.themes/default/source/arcs.html

```diff
@@ -2,20 +2,32 @@
 layout: page
 title: Arc Index
 rp_arcs: true
----
-
+--- 
+    
 <ul class="rp-list">
 {% for x in page['rps'] %}
   {% assign type = x[0] %}
   {% assign rp = x[1] %}
   {% if type == 'arc' %}
     <h3 class="arc-title"><a name="{{ rp.name | slugify }}" href="#{{ rp.name | slugify }}">{{ rp.name }}</a></h3>
+    <p class="arc-description">
+    {% for arc_rp in rp.rps %}
```

Comment:
This will behave oddly if different RPs in the same arc have descriptions. Example:
![image](https://user-images.githubusercontent.com/4063719/28253764-d0244c92-6a5b-11e7-951e-97a7ff28a9eb.png)

More comments below.